modules/ai/site-planner-connect/view.php

```diff
@@ -0,0 +1,175 @@
+		<g clip-path="url(#clip0_7635_41076)">
+			<path fill-rule="evenodd" clip-rule="evenodd"
+				  d="M257.022 26.6668C255.704 24.6934 255 22.3734 255 20C255 16.8174 256.264 13.7652 258.515 11.5147C260.765 9.26428 263.817 8 267 8C269.373 8 271.693 8.70379 273.667 10.0224C275.64 11.3409 277.178 13.2151 278.087 15.4078C278.995 17.6005 279.232 20.0133 278.769 22.3411C278.306 24.6688 277.164 26.807 275.485 28.4853C273.807 30.1635 271.669 31.3064 269.341 31.7694C267.013 32.2324 264.601 31.9948 262.408 31.0865C260.215 30.1783 258.341 28.6402 257.022 26.6668ZM264 14.9996H262.001V24.9999H264V14.9996ZM271.999 14.9996H266V16.9993H271.999V14.9996ZM271.999 18.999H266V20.9987H271.999V18.999ZM271.999 23.0002H266V24.9999H271.999V23.0002Z"
+				  fill="#0C0D0E"/>
+		</g>
+		<rect width="40" height="40" rx="20" fill="#F3F3F4"/>
+		<path fill-rule="evenodd" clip-rule="evenodd"
+			  d="M20.0004 10.0156C14.4944 10.0156 10.0156 14.494 10.0156 19.9996C10.0156 25.5053 14.4944 29.9844 20.0004 29.9844C25.5056 29.9844 29.9844 25.5053 29.9844 19.9996C29.9844 14.4947 25.5056 10.0156 20.0004 10.0156ZM11.1616 19.9996C11.1616 18.7184 11.4367 17.5017 11.927 16.4031L16.1431 27.9539C13.1948 26.5215 11.1616 23.4984 11.1616 19.9996ZM20.0004 28.8387C19.1327 28.8387 18.2954 28.7106 17.5032 28.4785L20.1549 20.7731L22.8725 28.2154C22.8898 28.2589 22.9115 28.2992 22.9353 28.3372C22.0167 28.6607 21.0292 28.8387 20.0004 28.8387ZM21.218 15.856C21.7501 15.8279 22.2293 15.7715 22.2293 15.7715C22.7058 15.7153 22.65 15.0158 22.1733 15.0438C22.1733 15.0438 20.7415 15.156 19.8176 15.156C18.9495 15.156 17.4894 15.0438 17.4894 15.0438C17.0133 15.0158 16.9579 15.744 17.4336 15.7715C17.4336 15.7715 17.8845 15.8277 18.3602 15.856L19.7373 19.6286L17.8034 25.4297L14.5851 15.8564C15.1178 15.8283 15.5968 15.7721 15.5968 15.7721C16.0725 15.7159 16.0169 15.016 15.54 15.0445C15.54 15.0445 14.1088 15.1564 13.1843 15.1564C13.0178 15.1564 12.823 15.1521 12.6157 15.1457C14.1954 12.7459 16.9123 11.1617 20.0004 11.1617C22.3018 11.1617 24.3964 12.0416 25.9689 13.4816C25.9302 13.4797 25.8937 13.4748 25.854 13.4748C24.9861 13.4748 24.3695 14.2309 24.3695 15.0434C24.3695 15.7715 24.789 16.388 25.2377 17.1159C25.5741 17.7051 25.9662 18.4613 25.9662 19.5537C25.9662 20.3102 25.6758 21.1882 25.2936 22.4107L24.4121 25.3566L21.218 15.856ZM24.4435 27.6389L27.1431 19.8337C27.6481 18.573 27.8152 17.5647 27.8152 16.6679C27.8152 16.343 27.7937 16.0404 27.7557 15.7591C28.4466 17.018 28.8391 18.4629 28.8386 19.9998C28.8386 23.2602 27.0708 26.1068 24.4435 27.6389Z"
+			  fill="#0C0D0E"/>
+		<defs>
+			<clipPath id="clip0_7635_41076">
+				<rect width="24" height="24" fill="white" transform="translate(255 8)"/>
+			</clipPath>
+		</defs>
+	</svg>
+
+	<div class="site-planner-consent-connect-names">
+		<div>%domain%</div>
+		<div>%app_name%</div>
+	</div>
+	<div style="height: 40px"></div>
+
+	<button class="site-planner-consent-button" onclick="sendPassword()">
+		%cta%
+	</button>
+</div>
+<script>
```

Comment:
Maybe use `wp_print_inline_script_tag()`, to make sure the script can still execute when a [CSP](https://content-security-policy.com/) header is present.

Reply:
How will it be different?

Reply:
@matipojo When it runs through `wp_print_inline_script_tag()` it will allow the user to add a filter to add some attributes to the script tag. But when you hard code the script tag, they cannot do this. If there is a CSP header present, but not the correct atts on your script tag, the browser will ignore this script.

Reply:
This script should be as is; we prefer not to allow filtering it. In case of a CSP issue, we will solve it ad-hoc.

Reply:
@matipojo Okay it's fine. But in general it's not good practice to have hard coded `<script>` tags. We are definitely going to come back here and fix it when we will make Elementor compatible with CSP. So it's kind of adding just one more to do item :)

Reply:
Here it's a sensitive script, so we will need to figure out what can be dynamic.

Reply:
Ok, thank you Mati.